.github/workflows/ci.yml

```diff
@@ -214,7 +214,7 @@ jobs:
         # flake8 should run on each Python version that we target,
         # because the errors and warnings can differ due to language
         # changes, and we want to catch them all.
-        python_version: ['3.6', '3.7']
+        python_version: ['3.7', '3.8']
```

Comment:
Want to add 3.9 and 3.10 in addition to 3.8?  We should be 3.10 compatible as of https://github.com/tensorflow/tensorboard/pull/5793, and while obviously we haven't been good about keeping this line up to date, the comment above has a rationale for running at least flake8 for every supported Python version.

If you do that we could also add 3.9 and 3.10 tags in both setup.py files as well.

Reply:
Good catch, updated!